Readability.js

```diff
@@ -1854,6 +1854,12 @@ Readability.prototype = {
     // get site name
     metadata.siteName = jsonld.siteName || values["og:site_name"];
 
+    // get image thumbnail
+    metadata.image = values["og:image"] || values.image || values["twitter:image"];
+
+    // get favicon
+    // metadata.favicon = this._getArticleFavicon()
```

Comment:
Can we drop this change? It doesn't seem to be doing anything.

Reply:
I just pushed another change, containing an extraction for favicons.